README.md

```diff
@@ -177,6 +177,51 @@ for job in batch.ordered_jobs:
     print(f"job-id: {job.id}, status: {job.status}, result: {job.result}")
 ```
 
+### Get a list of batches
+
+It is possible to get all batches or a selection of batches with the `get_batches` method.
+This method uses a pagination system that you have to handle.
```

Comment:
"that you have to handle" reads weird to me.

Suggestion:
The method returns by default a page of the 100 most recent batches but can be configured using the `pagination_params` argument